specifications/M1/verifiable.presentation.protocol.md

```diff
@@ -166,19 +176,19 @@ the [OpenID for Verifiable Presentations specifications](https://openid.net/spec
 
 # 5. Storage API
 
-VCs can be written to the Credential Service by POSTing an array of [Verifiable Credentials]() to
-the `credentials` endpoint
+VCs can be written to the Credential Service by POSTing a `CredentialMessage` containing an array
+of [Verifiable Credentials]() to the `credentials` endpoint:
 
 `POST /credentials`
 
 If the POST is successful, credentials will be created and a HTTP `2XX` is returned.
 
-The POST body is a JSON object with the following properties:
+The POST body is a `CredentialMessage` JSON object with the following properties:
 
 - `@context`: REQUIRED. Specifies a valid [Json-Ld context](https://www.w3.org/TR/json-ld11/#the-context).
-- `@type`: REQUIRED. A string specifying the credential type.
-- `credentials`: REQUIRED. A Json structure corresponding to the schema
-  specified [below](#the-credential-object).
+- `@type`: REQUIRED. A string specifying the `CredentialMessage` type.
+- `credentials`: REQUIRED. An array of `CredentialContainer` Json objects corresponding to the schema
```

Comment:
The concept of CredentialContainer is interesting, what is the motivation behind it? (Just out of interest for the topic).

Reply:
It's a holder for the actual credential content, which has to be defined as an XSD string since the credential may be in non-JSON form. Using a container as opposed to an array of strings allows for future extensibility in case additional attributes are needed.